docs/export-queue.md

```diff
@@ -0,0 +1,214 @@
+# Export Queue Recipe
+
+## Background
+
+Fluo is not suited for servicing low latency queries for two reasons.  The
+first reason is that the implementation of transactions are designed for
+throughput.  To get throughput transactions recover lazily from failures and
+may wait on another transaction thats writing.  Both of these design decisions
+can lead to delays for an individual transaction, but do not negatively impact
+throughput.   The second reason that Fluo observers executing transactions
+will likely cause a large number of random accesses.  This could lead high
+response time variability for an individual random access.  This variability
+would not impede throughput but would impede the goal of latency.
+
+One way to make data transformed by Fluo available for low latency queries is
+to export that data to another system.  For example Fluo could be running
```

Comment:
Could mention somewhere in the paragraph that the other system doesn't have to be Accumulo but could be redis, elasticsearch, MySQL, etc.